lib/puppet/pops/loader/static_loader.rb

```diff
@@ -108,6 +107,26 @@ def loaded_entry(typed_name, check_dependencies = false)
     @loaded[typed_name]
   end
 
+  def create_built_in_puppet_types
+    Pcore.add_object_type('Error', <<-PUPPET, self)
+      {
+        type_parameters => {
+          kind => Optional[Variant[String,Regexp,Type[Enum],Type[Pattern],Type[NotUndef],Type[Undef]]],
+          issue_code => Optional[Variant[String,Regexp,Type[Enum],Type[Pattern],Type[NotUndef],Type[Undef]]]
+        },
+        attributes => {
+          message => String[1],
+          kind => { type => Optional[String[1]], value => undef },
+          issue_code => { type => Optional[String[1]], value => 'ERROR' },
```

Comment:
How did we arrive at always having an issue_code and that the default should be 'ERROR' - that instead of just not having an issue code?

Reply:
I don't think that the issue_code 'ERROR' adds any value and just makes matching more confusing.